<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="excelBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select the Excel spreadsheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="functionBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Extract words</string>
    </property>
   </widget>
   <widget class="QPushButton" name="folderBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select output folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>721</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>No file selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="folderLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>No folder selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
